packages/transmute-cli/src/commands/provision/index.js

```diff
@@ -1,107 +1,119 @@
 const run = require('../runner');
 const logger = require('../../logger');
-const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.9.0';
+
+const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.10.0';
```

Comment:
Change to Kubernetes 1.10.0.  Kubernetes 1.9.4 was causing a failure to schedule minikube addons "storage-provisioner" and "dashboard" on macOS OS X 10.13.